terms/hyperparameters_nm.tex

```diff
@@ -0,0 +1,70 @@
+\chapter[Hyperparameters]{Hyperparameters}
+\label{ch:Hyperparameters}
+
+As stated by Li \textit{et al.} in \cite{Li2017}, "performance of machine learning algorithms depends critically on identifying a good set of hyperparameters". Hyperparameter tuning is a topic often discussed in literature describing any combination of machine learning algorithms. Adequate values set for hyperparameters will massively influence training of a model, be it in terms of processing time, as well as the actual robustness of the model in question. 
+
+\section{Introduction}\label{sec:page-layout}
+\subsection{Defining Hyperparameters}\label{sec:headings}\index{headings}
```

Comment:
WHat do you mean with \label{sec:headings}\index{headings} ??? Indexing "headings" ???? Meaningless.